databases/functions/userBlockUser/user-block.js

```diff
@@ -38,6 +39,8 @@ module.exports = async (
   const {
     message = '',
     postId = null,
+    // eslint-disable-next-line no-unused-vars
+    commentId = null,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similar to the previous comment, if `commentId` is not being used, it would be better to remove it rather than disabling the linter warning. However, if it's part of a function signature and can't be removed, then the ESLint disable comment is justified.
```diff
-     // eslint-disable-next-line no-unused-vars
-     commentId = null,
+     // commentId removed as it's not used
```
If `commentId` is expected to be used in future implementations, consider adding a comment explaining why it's there but not used. This will help other developers understand your intention.


<!-- This is an auto-generated comment by OSS CodeRabbit -->